src/main/java/com/checkout/payments/ProcessingSettings.java

```diff
@@ -13,4 +13,8 @@
 
     private boolean aft;
 
+    private Long taxAmount;
```

Comment:
Can you add @Serialized annotation for both new properties?

Reply:
Sure!